fpga.chart.py

```diff
@@ -187,7 +192,8 @@ def get_data(self):
 
         try:
             data = copy.deepcopy(self.default_data)
-            self.set_fpga_os_status(data)
+            if self.check_temp_power is True:
```

Comment:
You can omit the `is True`

Reply:
I think because we are reading it from a file it might be better to be explicit.

Reply:
Fine by me, was just a thought wrt Python best practices.

Reply:
ok